### ISSUE TITLE
Update a getting started message to be more accurate

### ISSUE THREAD

**Issue by jwren** [Dart-Code/Dart-Code#3819]
In the current getting started flow, the directions still say something along the lines of "connect a device and press F5 to launch."

With Chrome though as the default selection, I was able to just launch it without having to figure out how to connect a device.

@DanTup

@inmatrix for message verbage

**Comment by DanTup** [comment]
Yeah, perhaps this text was a bit misleading anyway as I suspect a lot of dev is done with emulator/simulator.

We could highlight where you can choose between devices ("Select a device in the status bar"), although I suspect that status bar entry is going to be tweaked in an upcoming release (VS Code is shipping some changes to stop the status bars becoming so full, so it's likely we'll have a fly-out menu bundling all of our status entries together).

Current text is here:

https://github.com/Dart-Code/Dart-Code/blob/3c7ff14939bb366e04c0b6db5e0c9d8151653f35/src/extension/user_prompts.ts#L224-L227

We could just drop the device part and use "Press F5 to start running" (debugging?). @InMatrix any opinions?

**Comment by DanTup** [comment]
I'm removing the "connect a device" so this will just say "Press F5 to start running". If there are no devices (eg. you don't have Chrome) it'll automatically open the device selector prompting for a device anyway, and if you connect a mobile device it'll automatically switch to that by default.

I can trivially revise the text again if there are better suggestions :-)
